src/skills-builder/skills-builder-modal/SkillsBuilderModal.jsx

```diff
@@ -48,8 +48,10 @@ const SkillsBuilderModal = () => {
           </ModalDialog.Hero.Content>
         </ModalDialog.Hero>
 
+        <Stepper.Header />
+
         <ModalDialog.Body>
-          <Container size="lg">
+          <Container size="md">
             <Stepper.Step eventKey={STEP1} title="Select preferences">
```

Comment:
This caught my eye but I know it's not part of this PR -- is the `title` displayed anywhere? Should this suport i18n/translation?

Reply:
Yes it does. Nice catch! The title attribute is the same text used to display in the `Stepper.Header`